src/main/python/gui/modulespecification_dialog.py

```diff
@@ -64,13 +65,23 @@ def __init__(self, moduletype, xslotstructure=None, moduletoload=None, linkedfro
         if isinstance(incl_articulators, str):
             incl_articulators = [incl_articulators]
 
+        if HAND in incl_articulators and self.parent().sign.signtype:
+            # set default articulators
+            handSelection = self.parent().sign.signtype.specslist[0][0]
+            if "1h" in handSelection:
```

Comment:
If we're ok with this logic, I thought of making `1h` and `2h` a constant string in the constants file. Not sure if that would be helpful elsewhere though?

Reply:
and we're not doing an exact match bc some of the options have more to the string than just `1h` (which should be irrelevant for my use case)